modules/router/src/listener.ts

```diff
@@ -822,6 +822,226 @@ export async function setupListeners(
       );
     }
   });
+  // TODO Refactoring, repeated code
+  await messagingService.onReceiveStartAuction(routerSigner.publicIdentifier, async (request, from, inbox) => {
```

Comment:
To be changed. I used code from other listeners. Would depend on new methods to calculate fees/swapRates